src/System.Runtime.Analyzers/UnitTests/NormalizeStringsToUppercaseTests.cs

```diff
@@ -9,12 +11,332 @@ public class NormalizeStringsToUppercaseTests : DiagnosticAnalyzerTestBase
     {
         protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
         {
-            return new BasicNormalizeStringsToUppercaseAnalyzer();
+            return new NormalizeStringsToUppercaseAnalyzer();
         }
 
         protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
         {
-            return new CSharpNormalizeStringsToUppercaseAnalyzer();
+            return new NormalizeStringsToUppercaseAnalyzer();
         }
+
+        #region No Diagnostic Tests
+
+        [Fact]
+        public void NoDiagnostic_ToUpperCases()
+        {
+            VerifyCSharp(@"
+using System;
+using System.Globalization;
+
+public class UnexpectedWarningAttribute : Attribute
```

Comment:
I recommend against that. I suspect there was something in the FxCop test infrastructure that used these constructs to emit more helpful error messages if the tests failed. But whether or not that's the case, these lines of code (the attribute classes and the `Console.WriteLine(this)` calls have no function in our test environment; they just make the test code bigger and harder to understand, and they make poor saps like me spend time trying to understand what they're for.